lib/Doctrine/DBAL/Portability/Connection.php

```diff
@@ -4,9 +4,11 @@
 
 use Doctrine\DBAL\Cache\QueryCacheProfile;
 use Doctrine\DBAL\ColumnCase;
+use Doctrine\DBAL\Driver\PDOConnection;
```

Comment:
Unrelated to this patch, but do we still need the portability connection in 3.x?

Reply:
What's wrong with it? The column case conversion is very helpful when dealing with Oracle and IBM DB2.

Reply:
On the other hand, I agree that `PORTABILITY_RTRIM` is just a crutch for badly designed code which happens to work on MySQL. And as for `PORTABILITY_EMPTY_TO_NULL` — no idea what it could be used for.